### PR TITLE
Guess username renames using GitHub redirects in ghostbuster

### PR DESCRIPTION
**Pull request by jakebailey**
We already manually try and figure out renames via googling people's GitHub profile URLs and trying to find repos that redirect.

If we assume that users will have probably forked DefinitelyTyped itself at some point, we can `fetch` that URL and see if it redirects. If it does, we have the new username.

I'm not totally sure if this is a good idea to do automatically; I know some people may change their usernames for personal reasons and would rather not directly have this rename visible, but we already manually ping people so it's theoretically no worse.

### CODE REVIEW

**Comment by andrewbranch** [comment]
I think doing the _pinging_ automatically is ok, but updating the username automatically might be too presumptive. Here’s the saved reply I use, if you feel like changing this to a ping from the bot:

> @{new name} if you’d still like to be an owner for these types, please feel free (but no pressure) to add your new account name back to the header. Thanks!

In my experience doing this (a fairly small sample size):

- most have not responded, and AFAIK not added themselves back
- at least one responded positively and added themselves back
- nobody has responded negatively

**Comment by jakebailey** [comment]
The gotcha is that the PR reply is generated in the markdown and the script just modifies the files; but I could have it print to stdout the markdown and generate it. I just avoided it from the get-go.

**Comment by jakebailey** [comment]
Updated to instead put the pings in a PR description.

File: scripts/ghostbuster.js [hunk 1]
```diff
@@ -86,9 +86,9 @@ function getAllPackageJsons() {
  * @param {Set<string>} users
  */
 async function fetchGhosts(users) {
-    console.log("Checking for deleted accounts...");
+    console.error("Checking for deleted accounts...");
     const octokit = new Octokit({ auth: process.env.GITHUB_TOKEN });
-    const maxPageSize = 2000;
+    const maxPageSize = 500;
```
Comment (jakebailey) [inline scripts/ghostbuster.js:91]: Drive-by, but 2000 consistently caused GitHub to 503 on my machine. It always works in CI, though.

**Review by andrewbranch — APPROVED** [review]

File: .github/workflows/ghostbuster.yml [hunk 2]
```diff
@@ -61,7 +61,4 @@ jobs:
           branch-suffix: short-commit-hash
           delete-branch: true
           title: Remove contributors with deleted accounts
-          body: |
-            Generated from [.github/workflows/ghostbuster.yml](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/.github/workflows/ghostbuster.yml)
-
-            Some of these users may have simply changed their usernames; you may want do a bit of searching and ping them to see if they still want to be owners.
+          body-path: comment.md
```
Comment (andrewbranch) [inline .github/workflows/ghostbuster.yml:64]: TIL. Super cool

**Comment by jakebailey** [comment]
Ready to merge

**Pull request merged**